configs/metadata.yaml

```diff
@@ -133,5 +167,20 @@ linz:
       red: 25
     wavelength:
       blue: 0.48
```

Comment:
Camera used for LINZ aerial surveys differs by council, but for https://data.linz.govt.nz/layer/115053-christchurch-0075m-urban-aerial-photos-2023/metadata/dc which used a Leica ADS100 line scanning camera, I got these specs off https://leica-geosystems.com/-/media/files/leicageosystems/products/datasheets/leica_ads100_ds.pdf?sc_lang=en-us&hash=8A2985763B14A56C278FA85CB5FF88C8.

![image](https://github.com/Clay-foundation/model/assets/23487320/c825b209-ca79-43e1-8458-8cb972d834fd)

```suggestion
      blue: 0.465
```